docs/content/docs/changelogs/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Working changelog
+draft: true
```

Comment:
```suggestion
draft: true # Don't publish this!
```

You can change this - It's just worth noting that the "draft" is intentional for the benefit of future folks.